runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldTableRow.java

```diff
@@ -64,12 +62,17 @@
 	private static final Color FREE_WORLD = new Color(200, 200, 200);
 	private static final Color LEAGUE_WORLD = new Color(157, 237, 1);
 
+	private static final Map<WorldRegion, ImageIcon> FLAGS;
+
 	static
 	{
-		FLAG_AUS = new ImageIcon(ImageUtil.getResourceStreamFromClass(WorldHopperPlugin.class, "flag_aus.png"));
-		FLAG_UK = new ImageIcon(ImageUtil.getResourceStreamFromClass(WorldHopperPlugin.class, "flag_uk.png"));
-		FLAG_US = new ImageIcon(ImageUtil.getResourceStreamFromClass(WorldHopperPlugin.class, "flag_us.png"));
-		FLAG_GER = new ImageIcon(ImageUtil.getResourceStreamFromClass(WorldHopperPlugin.class, "flag_ger.png"));
+		ImmutableMap.Builder<WorldRegion, ImageIcon> builder = new ImmutableMap.Builder<>();
+		for (WorldRegion worldRegion : WorldRegion.values())

```

Comment:
This type of thing becomes easy to miss later when adding a new world region. Keeping the previous code and changing getFlag to `getFlag(WorldRegion region)` can compile-time error due to the switch if you cover all possible enum values.

Reply:
Wouldn't it still throw an error since it couldn't find the file, if only an enum entry was added?

Reply:
If it was run yes, but I don't think this is even tested so we wouldn't notice.

Reply:
Added a test that would check for this